yarn.lock

```diff
@@ -5,46 +5,6 @@ __metadata:
   version: 6
   cacheKey: 8
 
-"@0no-co/graphql.web@npm:^1.0.1":
```

Comment:
here is a guide on how to remove the changes form yarn.lock: https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md#guidelines-for-committing-yarn-lockfile

Reply:
I'm very sorry again. The fix didn't work for me so I had to manually delete the file, add a new commit and open a new PR.

**PR: #14948**

Please have a look at it. Thanks!!

Reply:
Done. Deleted the yarn.lock file. Please check.
**PR: #14889**

Reply:
https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md#guidelines-for-committing-yarn-lockfile

This fix wasn't working for me, the yarn.lock changes were not reverting. That's why I had to delete the yarn.lock file.

Reply:
Closed this one and opened a new PR #14975.